### PR TITLE
Add circleci

### PR DESCRIPTION
**Pull request by Xorima**
Enabled Circle CI
Removed Travis

### CODE REVIEW

**Comment by Xorima** [comment]
Can someone take a look at the Dangerfile issue, I do not understand the api token issue

File: metadata.rb [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
+chef_version     '>= 12.7' if respond_to?(:chef_version)
 
 %w( debian ubuntu centos redhat oracle amazon ).each do |os|
   supports os
 end
 
 depends 'mysql'
 depends 'mysql2_chef_gem', '>= 2.0'
-depends 'compat_resource', '>= 12.16'
+# depends 'compat_resource', '>= 12.16'
```
Comment (damacus) [inline metadata.rb]: This can go now we're definitely on chef 13

**Comment by josephholsten** [comment]
@Xorima this looks quick, care to knock it out? or do you want a hand?

**Comment by Xorima** [comment]
I can do it at some point, away on holiday at the moment however so your call on how quickly you want it done?

**Comment by josephholsten** [comment]
no worries, was just going through open tickets at the meeting this morning and saw you were this `-><-` close

**Comment by Xorima** [comment]
As raised in the slack channel, I think we need to have a conversation about the usefullness and future of this cookbook vs just including anything of use into the mysql cookbook?

**Pull request merged**